src/containers/ElementConfig.tsx

```diff
@@ -0,0 +1,18 @@
+import { AppProvider } from 'lodestar-app-element/src/contexts/AppContext'
+import { AuthProvider } from 'lodestar-app-element/src/contexts/AuthContext'
+import React from 'react'
+import { useAuth } from '../contexts/AuthContext'
+
+export const ElementConfig: React.FC<{
```

Comment:
I think this is also a context, not a container(HOC)
maybe `ElementProvider` is a better naming